api/spec/src/billing/invoice/bill.tsp

```diff
@@ -0,0 +1,407 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+
+namespace Invoice.Bill;
+
+@summary("Invoice summary represents an extract of the invoice object")
+@friendlyName("BillingInvoiceSummary")
+model InvoiceSummary {
+  ...OpenMeter.InvoiceSummary; // OpenMeter schema extensions
```

Comment:
comment sounds out of context, I guess Gobl left over

Reply:
Nope, I intentionally left it there as per the considerations of [README.md](https://github.com/openmeterio/openmeter/pull/1641/files#diff-44d0a2bc7ca532a2725b5e4284ff129356b058f537ab2d2db519f2b8d3f0e729)

Reply:
But further thinking about your confusion, I start to see little value to signify the changes in gobl schema, so I have removed both these namespaces both the README.md